cibuild.sh

```diff
@@ -299,12 +299,18 @@ if [[ $BOOTSTRAP_ONLY = true ]]; then
     exit $?
 fi
 
+# Microsoft.Net.Compilers package is available now, so we can use the latest csc.exe
+if [ "$host" = "Mono" ]; then
+        CSC_EXE=`find $PACKAGES_DIR/microsoft.net.compilers -name csc.exe`
+        CSC_ARGS="/p:CscToolExe=csc.exe /p:CscToolPath=`dirname $CSC_EXE` /p:DebugType=portable"
+fi
+
 # The set of warnings to suppress for now
 # warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.
 # warning MSB3026: Could not copy "XXX" to "XXX". Beginning retry 1 in 1000ms.
 # warning AL1053: The version '1.2.3.4-foo' specified for the 'product version' is not in the normal 'major.minor.build.revision' format
-_NOWARN="MSB3277;MSB3026;AL1053"
+_NOWARN="MSB3276;MSB3277;MSB3026;AL1053"
```

Comment:
References which depend on "mscorlib, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" [/Library/Frameworks/Mono.framework/Versions/4.8.0/lib/mono/2.0-api/mscorlib.dll]. (TaskId:188)

/Users/ankit/.nuget/packages/Microsoft.VisualStudio.Setup.Configuration.Interop/1.2.304-preview5/lib/net35/Microsoft.VisualStudio.Setup.Configuration.Interop.dll (TaskId:188)
            Project file item includes which caused reference "/Users/ankit/.nuget/packages/Microsoft.VisualStudio.Setup.Configuration.Interop/1.2.304-preview5/lib/net35/Microsoft.VisualStudio.Setup.Configuration.Interop.dll". (TaskId:188
)

/Users/ankit/.nuget/packages/Microsoft.VisualStudio.Setup.Configuration.Interop/1.2.304-preview5/lib/net35/Microsoft.VisualStudio.Setup.Configuration.Interop.dll (TaskId:188)

---

So, this is coming from `Microsoft.VisualStudio.Setup.Configuration.Interop.dll`. What would be a better fix:

1. ignore the warning
2. add remapping in app.config (?)
3. Set `$(AutoGenerateBindingRedirects)=true` in the csproj